langchain4j-vearch/src/test/java/dev/langchain4j/store/embedding/vearch/TestVearchClient.java

```diff
@@ -0,0 +1,61 @@
+package dev.langchain4j.store.embedding.vearch;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import lombok.Builder;
+import okhttp3.OkHttpClient;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import static com.google.gson.FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES;
+
+public class TestVearchClient {
```

Comment:
Why can't you use real client for testing?

Reply:
`TestVearchClient` only aims to support `clearStore` in IT(delete space), which is not included in real client.

Reply:
Makes sense, thanks for explanation!